xnat-data-structure-requirements-specification.md

```diff
@@ -4,20 +4,80 @@
 
 ...
 
-**Goal**: give a structured description and insight in how non-raw DICOM and non-DICOM data (e.g., converted DICOM, segmentations, derived data) in a project is structured, what the files contain, and allow users to expose this structure through the API to be used with tooling like [xnatpy](https://xnat.rtfd.io). 
+**Goal**: give a structured description and insight in how non-raw DICOM
+and non-DICOM data (e.g., converted DICOM, segmentations, derived data) 
+in a project is structured, what the files contain, and allow users to
+expose this structure through the API to be used with tooling like 
+[xnatpy](https://xnat.rtfd.io). 
+
+## Motivation
+
+The XNAT medical image storage solution has defined a data structure for
+medical imaging data. This is following the concepts/levels of `project`,
+`subject`, `experiment`, `scan`, and `resource`. This gives the data it's
+general structure. The names of each level and the contents are not
+pre-defined. This allows flexibility for users to store any kind of data in
+the archive and match their use-case, but also makes the data hard to interpret
+without additional information. Getting access to a project on XNAT will not
+make you understand the data further than the general structure.
+
+For data to be FAIR the data should be well described so that new users can
+understand and use the data. This requires a more comprehensive description of
+the data in an archive. For this we propose to create a *data-structure file*
+for each project on XNAT, which describes what data is available for that 
+project.
+
+Such a file will not only allow people to immediately know what data is in a 
+project, but also allow the creation of tools to help with data-management. 
+These tools could include tools to validate data, select and process data, 
+storing data under the right place automatically and more.
```

Comment:
I would add that the formalizing the data structure is actually a form meta-data definition, this will allow for direct interaction with catalogues. For instance, the aggregates could be collected and exposed through some means (fdp/dcat, or some other standard that seems the most relevant) to for instance the Health-RI catalog or any other catalog for that matter.